src/prefect/client/schemas/actions.py

```diff
@@ -160,6 +161,11 @@ class DeploymentCreate(ActionBaseModel):
     def remove_old_fields(cls, values):
         return remove_old_deployment_fields(values)
 
+    @field_validator("description", "tags", mode="before")
+    @classmethod
+    def convert_to_strings(cls, values):
+        return convert_to_strings(values)
+
```

Comment:
It's surprising we need to do this, what were `description` and `tags` coming through as before?

Reply:
v1 seems to have been converting things like `1` into `"1"` automatically, but not v2.